plugins/lua/design.lua

```diff
@@ -0,0 +1,15 @@
+local _ENV = mkmodule('plugins.design')
+
+function draw_shape(arr)
+    design_draw_shape(arr)
+end
+
+function draw_points(points_obj)
+    design_draw_points(points_obj)
+end
+
+function clear_shape(arr)
+    design_clear_shape(arr)
+end
```

Comment:
are these here just for the rename? otherwise, the plugin-exported functions are directly usable by other scripts. This file needs to exist, but could just be:
```lua
local _ENV = mkmodule('plugins.design')

return _ENV
```